lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -291,4 +300,24 @@ public function generate_styles() {
 
 		return $styles;
 	}
+
+	/**
+	 * Preload webfonts to improve performance
+	 *
+	 * @return void
+	 */
+	public function preload_webfonts() {
+		foreach ( $this->get_fonts() as $font ) {
+			if ( ! empty( $font['preload'] ) && $font['preload'] ) {
```

Comment:
This logic is deeply nested causing the code to shift to the right. This can decrease readability. I'd suggest either breaking it up into private methods -or- converting this guard into a skip clause like this:

```suggestion
			// Skip this font if it's not configured for preloading.
			if ( empty( $font['preload'] ) ) {
				continue;
			}
```